packages/slice-machine/src/features/onboarding/OnboardingProgressStepper.tsx

```diff
@@ -33,37 +35,47 @@ export const OnboardingProgressStepper = () => {
         isOpen={isDialogOpen}
         onClose={() => setDialogOpen(false)}
       />
-      <DropdownMenu>
+      <DropdownMenu open={isListOpen} onOpenChange={setListOpen}>
         <DropdownMenuTrigger>
           <Button
             color="grey"
             sx={{ width: "100%" }}
             renderEndIcon={EndCtaIcon}
+            onMouseEnter={() => setListOpen(true)}
           >
             {completedStepCount > 0 ? "Continue" : "Start"}
           </Button>
         </DropdownMenuTrigger>
-        <DropdownMenuContent align="start">
-          <DropdownMenuLabel>Progress</DropdownMenuLabel>
-          {steps.map((step) => {
-            const isCompleted = isStepComplete(step.id);
+        <DropdownMenuContent align="center" sideOffset={-56} minWidth={256}>
+          <div
+            onMouseLeave={() => {
+              if (!isListOpen) return;
```

Comment:
💡 i think you can avoid this as setting the state to the value it is already is a no-op.

Reply:
thanks, will remove that line!